### PR TITLE
NO-TICKET: Pass currentMillis explicitly when creating packets.

### PR DESCRIPTION
**Pull request by goral09**
## Overview
Provide a brief description of what this PR does, and why it's needed.

### Which JIRA issue does this PR relate to? If there is not a JIRA issue addressing this work, please create one now and add the link here.
Add link to corresponding JIRA issue.

### Complete this checklist before you submit the PR
- [x] This PR contains no more than 200 lines of code, excluding test code.
- [x] This PR meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards).
- [x] You have someone in mind to assign for review. Make this assignment after submitting the PR.

### Notes
Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else.

### CODE REVIEW

File: comm/src/main/scala/coop/rchain/comm/discovery/KademliaNodeDiscovery.scala [hunk 1]
```diff
@@ -2,29 +2,32 @@ package coop.rchain.comm.discovery
 
 import scala.collection.mutable
 import scala.concurrent.duration._
+import cats.effect.Timer
```
Comment (birchmd) [inline comm/src/main/scala/coop/rchain/comm/discovery/KademliaNodeDiscovery.scala]: Unused import?

File: comm/src/main/scala/coop/rchain/comm/rp/Connect.scala [hunk 2]
```diff
@@ -1,21 +1,21 @@
 package coop.rchain.comm.rp
 
-import coop.rchain.p2p.effects._
+import cats._
+import cats.effect.Timer
```
Comment (birchmd) [inline comm/src/main/scala/coop/rchain/comm/rp/Connect.scala]: And here

**Comment by goral09** [comment]
@rabbitonweb I asked that question on comm channel on gitter. Method creating packets' header is not pure and it's impossible to test what get's send to the peers. The only way to test is to inspect the logs. By passing the timestamp we can compare now test that we really send what we meant.

**Comment by rabbitonweb** [comment]
@goral09 as discussed a second ago, this parameter should be r3emoved from header, will do that in next PR
pls close this PR

**Pull request closed without merging**